autogenerate_api_client.sh

```diff
@@ -0,0 +1,18 @@
+curl -X POST https://generator3.swagger.io/api/generate -H 'content-type: application/json' -d '{ "specURL" : "https://tdei-gateway-dev.azurewebsites.net/v3/api-docs", "lang" : "typescript-axios", "type" : "CLIENT", "codegenVersion" : "V3"}' --output tdei-client-axios.zip
```

Comment:
This curl request does not specify the name of the repo. Needs to be modified. Eg.

```shell
curl --location 'https://generator3.swagger.io/api/generate' \
--header 'Content-Type: application/json' \
--data '{
    "specURL": "https://tdei-gateway-dev.azurewebsites.net/v3/api-docs",
    "lang": "typescript-axios",
    "type": "CLIENT",
    "codegenVersion": "V3",
    "options": {
        "additionalProperties": {
            "modelPropertyNaming": "original",
             "npmName":"tdei-client",
             "npmVersion":"1.0.0"
        }
    }
}' --output tdei-client.zip
```

Reply:
Even in the above code, there should be a way to specify the npmVersion from user or auto increment

Reply:
Will see if I can read the npm version of the previous package.json and autoincrement it?

Reply:
> This curl request does not specify the name of the repo. Needs to be modified. Eg.
> 
> ```shell
> curl --location 'https://generator3.swagger.io/api/generate' \
> --header 'Content-Type: application/json' \
> --data '{
>     "specURL": "https://tdei-gateway-dev.azurewebsites.net/v3/api-docs",
>     "lang": "typescript-axios",
>     "type": "CLIENT",
>     "codegenVersion": "V3",
>     "options": {
>         "additionalProperties": {
>             "modelPropertyNaming": "original",
>              "npmName":"tdei-client",
>              "npmVersion":"1.0.0"
>         }
>     }
> }' --output tdei-client.zip
> ```

You can replace it with this @santaduda. The one I gave you also works but specifying the additional properties while generating the client makes sense as we initially did the same